src/blog/2019-04-03/one-mutation-testing-html-report.pug

```diff
@@ -0,0 +1,98 @@
+        During development of Stryker4s and Stryker.NET, we realized that producing the same report would be a lot of
+        work. 
+        
+        Instead of rebuilding the generating of HTML files for each Stryker implementation, we've decided to move the logic of presenting the report to your browser.
+        That way, any framework can support an HTML report, simply by producing a JSON file.
+        We call it [mutation testing elements](https://github.com/stryker-mutator/mutation-testing-elements#readme) and 
+        it is implemented using the mature
+        [web components suite of features](https://www.w3.org/TR/components-intro/).
+
+        You can now use it in Stryker, Stryker.NET and Stryker4s
+
+        ## 👷‍ Usage
+
+        To enable the new HTML reporter, simply update to the latest Stryker release and 
+        enable the HTML reporter in the implementation of your choice.
+
+        * **Stryker** (since v1.2)  
+        Enable by adding `'html'` to your `reporters` in your `stryker.conf.js` file.
+        * **Stryker4s** (since v0.3.1)  
+        Enable by adding `"html"` to your `reporters` in your `stryker4s.conf` file.
+        * **Stryker.NET** (since v0.9)  
+        Enable `"html"` as reporter in your `stryker-config.json` file.
+
+        Upgrading should be painless.
+        
+        ## 💸 Show me the money
+
+        See two screenshots taken from a mutation testing report of Stryker itself 🎉
+
+        ![Directory](/images/blogs/mutation-testing-elements-directory.png)
+
+        ![File](/images/blogs/mutation-testing-elements-file.png)
+
+        It might look pretty familiar, that's because it is based on the old Stryker implementation.
+        
+        Mutation testing elements comes with batteries included:
+        
+        📊 Calculates and displays the mutation score and other metrics  
+        📁 Group results into directories  
+        👓 Show mutants directly in your source code  
+        😎 Code highlighting with [highlightjs](https://highlightjs.org/)  
+        🧙‍ Filter mutants based on the outcome  
+        🔗 Deep linking using anchors (uses fragment, so path will not be contaminated)  
+        ✨ A nice (yet basic) UI with embedded [bootstrap](https://getbootstrap.com) css  
+        🎓 Adheres to [custom element best practices](https://developers.google.com/web/fundamentals/web-components/best-practices)
+
+        ## 💭 How it works
+
+        Using mutation testing elements is as simple as loading the JavaScript bundle and adding 
+        the `mutation-test-report-app` to your page, pointing the `src` to a json file that contains the 
+        mutation testing results to be displayed.
+
+        ```html
+        <body>
+        <script src="https://www.unpkg.com/mutation-testing-elements@1.0.2/dist/mutation-test-elements.js"></script>
+        <mutation-test-report-app src="mutation-report.json">
```

Comment:
Is having the script loaded from unpkg and the report in a separate file the 'recommended' way? I thought that had some issues. I think we should have the 'best' way to do things as an example

Reply:
I would say `src="mutation-report.json"` is preferred, except when opening a file locally, as it is not supported by chrome. Do we want to detail that in this blog article?

Reply:
Maybe. I think most users will open the file locally, after a mutation run. So any integrators should not do it this way

Reply:
I've changed the wording to clarify that property binding should be to support opening it from disk locally